products/rtpengine.md

```diff
@@ -0,0 +1,48 @@
+---
+title: rtpengine 
+category: server-app
+permalink: /rtpengine
+versionCommand: rtpengine -v
+releaseDateColumn: true
+
+identifiers:
+-   repology: rtpengine
+auto:
+  methods:
+    - git: https://github.com/sipwise/rtpengine.git
+      regex: ^mr(?P<major>\d+)_(?P<minor>\d+)_(?P<patch>\d{1,3})_?(?P<tiny>\d+)?$
+
+releases:
+
+-   releaseCycle: "mr13.0"
+    releaseDate: 2024-10-01
+    eol: true
+    link: https://www.sipwise.com/wp-content/uploads/2024/11/Changelog_for_mr13.0.1_20241119.pdf
+    latest: "mr13.0.1"  
+
+-   releaseCycle: "mr12.5"
+    releaseDate: 2024-08-23
+    eol: false
```

Comment:
EOL dates can be found on https://www.sipwise.com/releases/releases.json. Can you set them for each release.

Would be nice to use this for automation after this is merged.

Reply:
done.

Reply:
https://github.com/endoflife-date/release-data/issues/413